fix(api): clarify Window scroll options (#44116)
mdn/content: files/en-us/web/api/window/scrollby/index.md

- `options`
  - : An object containing the following properties:
    - `top`

>       - : Specifies the number of pixels along the Y axis to scroll the window or element.

Reviewer: I think the original wording was fine in these cases. In your new wording, "to scroll by along" is not grammatically good. Revert these two changes?

Author: Thanks, agreed. I reverted these two descriptions to the original wording.

    - `left`
      - : Specifies the number of pixels to scroll by along the X axis.
    - `behavior`

>       - : Determines whether scrolling is instant or animates smoothly. This option is a string which must take one of the following values:

Reviewer: ```suggestion
      - : Determines whether the scrolling is instant or animates smoothly. This option is a string that must take one of the following values:
```

Author: Thanks, applied.

>         - `smooth`: scrolling should animate smoothly

Reviewer: ```suggestion
        - `smooth`: The scrolling animates smoothly.
```

>         - `instant`: scrolling should happen instantly in a single jump

Reviewer: ```suggestion
        - `instant`: The scrolling happens instantly in a single jump.
```

>         - `auto`: scroll behavior is determined by the computed value of {{cssxref("scroll-behavior")}}

Reviewer: ```suggestion
        - `auto`: The scroll behavior is determined by the computed value of the {{cssxref("scroll-behavior")}} CSS property on the document.
```

Reviewer: Maybe also make these changes in `scrollTo`?

Also, are some of the problems you are trying to fix present on the `Element.*` versions of these pages, too?
